cmd/kyma/provision/minikube/cmd.go

```diff
@@ -70,6 +70,7 @@ func NewCmd(o *Options) *cobra.Command {
 	cmd.Flags().StringVar(&o.CPUS, "cpus", "4", "Specifies the number of CPUs used for installation.")
 	cmd.Flags().StringVar(&o.Profile, "profile", "", "Specifies the Minikube profile.")
 	cmd.Flags().DurationVar(&o.Timeout, "timeout", 5*time.Minute, `Maximum time during which the provisioning takes place, where "0" means "infinite". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".`)
+	cmd.Flags().BoolVar(&o.UseVPNKitSock, "use-vpnkit-sock", false, `Uses vpnkit sock provided by Docker. This is useful when the DNS Port (53) is used by some other program like dns-proxy (e.g. provided by Cisco Umbrella).`)
```

Comment:
That is the minikube docu saying: "--hyperkit-vpnkit-sock='': Location of the VPNKit socket used for networking. If empty, disables Hyperkit
VPNKitSock, if 'auto' uses Docker for Mac VPNKit connection, otherwise uses the specified VSock (hyperkit driver only)"

Have you checked if you can have it on by default? Have you checked if it works with "auto", maybe even for all cases?
I'm thinking if we should maybe add "hyperkit" to the naming to build the bridge to the related minikube flag

Reply:
`auto` is looking for the socket at a different path so its not working. Updated the name of the flag